rootfs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/google_containers/ubuntu-slim:0.5
+FROM ubuntu
```

Comment:
I was just (mildly) concerned that we would be surprised by changes in the next LTS version `18.04`, although that's like a million years away. 😄